.docker/ci.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM jekyll/jekyll:4.0.0
+
+WORKDIR /srv/jekyll
+VOLUME  /srv/jekyll
+
+RUN mkdir -p /var/jekyll && \
+    mkdir -p /srv/jekyll && \
+    mkdir -p /srv/jekyll/_site && \
+    mkdir -p /srv/jekyll/.jekyll-cache
+
+RUN apk add --no-cache --no-progress\
+    graphviz\
+    openjdk8-jre-base\
+    fontconfig\
+    ttf-dejavu
+
+COPY . .
+
+# Work around a nonsense RubyGem permission bug.
+# RUN unset GEM_HOME && unset GEM_BIN && yes |
+RUN gem install --force bundler && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5414481)